test/include/dlaf_test/matrix/util_matrix_local.h

```diff
@@ -13,21 +13,22 @@
 /// @file
 
 #include <functional>
-#include <lapack/util.hh>
 #include <type_traits>
 
-#include <gtest/gtest.h>
+#include <lapack/util.hh>
```

Comment:
From a quick check, we can probably remove this include.

Apparently, it was introduced for `MatrixType` in https://github.com/eth-cscs/DLA-Future/pull/410, but then it became useless with the refactoring in https://github.com/eth-cscs/DLA-Future/pull/484